### DISCUSSION TITLE
Automatically installing plugins listed in the project.yaml file

### DISCUSSION THREAD

**Discussion by nmsdvid** [craftcms/cms#8615] [Ideas]
### Description
Automatically installing plugins listed in the project.yaml file. Currently you need to install the plugins manually, would be cool if Craft would installed it automatically when it detects there was a change in the project.yaml file.

**Comment by brandonkelly** [comment]
It already does :) Have you found a case where it does not?

**Comment by nmsdvid** [comment]
What I meant is if a plugin is listed in the project.yaml like this:
```
plugins:
  redactor:
    edition: standard
    enabled: true
    schemaVersion: 2.2.1
```
Craft would download and install the dependency automatically. Right now it says:

> The following plugins are listed in project.yaml, but appear to be missing or installed at the wrong version: redactor
> 
> Try running composer install from your terminal to resolve.

**Comment by brandonkelly** [comment]
Ah right, you want them to be Composer-installed. Yep that would be cool. Maybe we can get to this when we rewrite Craft’s self-updater soon.

**Comment by ca-miked** [comment]
+1

**Comment by pathei** [comment]
Any news on this one? @nmsdvid cheeres from cgn ;)
